alacritty_terminal/src/term/mod.rs

```diff
@@ -901,6 +902,11 @@ impl<T> Term<T> {
             }
         }
         self.visual_bell.update_config(config);
+        if config.scrolling.faux_multiplier().unwrap_or(config.scrolling.multiplier()) > 0 {
+            self.mode.insert(TermMode::ALTERNATE_SCROLL)
+        } else {
+            self.mode.remove(TermMode::ALTERNATE_SCROLL)
+        }
```

Comment:
Actually, I'm not sure this is needed. `TermMode::ALTERNATE_SCROLL` is enabled by default and even if multiplier is 0, scrolling won't work.